config/config_test.go

```diff
@@ -206,6 +206,22 @@ func TestGetAsIntegerValue(t *testing.T) {
 	}
 }
 
+func TestGetAsFloatValue(t *testing.T) {
+	testCases := []struct {
+		value interface{}
+	}{
+		{float32(2)},
+		{float64(2)},
+		{int(2)},
```

Comment:
Good question. Though about this too but not sure if config values can be sent as unsigned types. We might want to do TryAsUnsignedInt but that's different from what you're asking.
What do you think @anuptalwalkar